package.json

```diff
@@ -2,16 +2,20 @@
     "name": "coveo-search-ui-extensions",
     "version": "0.0.2",
     "description": "Small generic components to extend the functionality of Coveo's Search UI framework.",
-    "main": "./bin/js/CoveoJsSearchExtensions.js",
+
+    "main": "./bin/js/index.js",
+    "types": "./bin/typings",
+
+    "sideEffects": false,
+    
     "scripts": {
-        "test": "karma start --single-run",
+        "test": "karma start ./config/karma.config.js --single-run",
         "testCoverage": "npm run test && cat ./bin/coverage/lcov.info | node ./node_modules/coveralls/bin/coveralls.js",
```

Comment:
You need to change the path here for `../node_modules/`,  I think that's why coveralls didn't work?

Reply:
Will try but it does not makes sense to me. I mean, the package.json file is on root 😕 .

Reply:
oh you're right.. nevermind then :P

Reply:
ah, make sure the coverage is output in the right folder, in the karma.config.json, it outputs to `./bin/coverage`, pretty sure that will be in the config folder then

Reply:
Oups 😱